api/types/src/account.rs

```diff
@@ -1,22 +1,96 @@
 // Copyright (c) Aptos
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{HexEncodedBytes, U64};
+use crate::{Address, HexEncodedBytes, U64};
 
-use aptos_types::account_config::AccountResource;
+use aptos_types::{
+    account_config::{AccountResource, TransferEvents as TransferEventsStruct},
+    event::EventKey,
+};
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct AccountData {
     pub sequence_number: U64,
     pub authentication_key: HexEncodedBytes,
+    pub self_address: Address,
+    pub balance: TestCoin,
+    pub transfer_events: TransferEvents,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct TestCoin {
+    pub value: U64,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct TransferEvents {
+    pub sent_events: EventHandle,
+    pub received_events: EventHandle,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct EventHandle {
+    pub counter: U64,
+    pub guid: EventHandleGUID,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct EventHandleGUID {
+    len_bytes: u8,
```

Comment:
The API types doesn't need to reflect the hack in Move? (conversion is manual anyway)

Reply:
I don't know how this thing works..

Reply:
Just remove the type? This means nothing to the client and user. Just use GUID in place of `EventHandleGUID`. We'll clean up on the Move side anyway, then it won't be a breaking change in the REST API then.